MYabrv.bib

```diff
@@ -58,21 +58,23 @@ @String{Informatik
 @String{ITNG = "Proc.\ Inte'l Conf.\ on Information Technology: New Generations (ITNG)"}
 @String{ITP = "Proc.\ Inte'l Conf.\ on Interactive Theorem Proving (ITP)"}
 @String{JMLC = "Proc.\ Joint Modular Languages Conference (JMLC)"}
+@String{MSR = "Proc.\ Working Conf.\ on Mining Software Repositories (MSR)"}
```

Comment:
the idea is to have alphabetical order, which was wrong before. please move all entries starting with M